examples/Paris_1872_Alexis/Paris_1872_Alexis.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../TEI_documentation/openStemmata.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../TEI_documentation/openStemmata.rng"" type="application/xml"
+	schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title type="standard">Stemma of</title>
+            <title type="variable">Chanson de saint Alexis</title>
+            <author>Paris, Gaston and Pannier, Léopold</author>
+            <respStmt>
+               <resp>contributed to OpenStemmata by</resp>
+               <persName>Jean-Baptiste Camps</persName>
+               <idno type="ORCID" n="0000-0003-0385-7037"/>
+            </respStmt>
+         </titleStmt>
+         <publicationStmt>
+            <authority>
+               <name>Openstemmata</name>
+               <ptr target="https://openstemmata.github.io/"/>
+            </authority>
+            <availability>
+               <licence n="cc.by" target="https://creativecommons.org/licenses/by/4.0"/>
+            </availability>
+         </publicationStmt>
+         <notesStmt>
+            <note>Deprecated stemma (see Zufferey’s for a more recent one)</note>
+         </notesStmt>
+         <sourceDesc>
+            <bibl type="book">
+               <title>La vie de saint Alexis : poème du XIe siècle et renouvellements des XIIe, XIIIe, et XIVe siècles</title>
+               <date>1872</date>
+               <pubPlace>Paris</pubPlace>
+               <series>Bibliothèque de l’École des hautes études. Sciences philologiques et historiques</series>
+               <!-- Numero de série -->
+               <biblScope unit="volume" n="7"/>
+               <author>Paris, Gaston and Pannier, Léopold</author>
+               <biblScope unit="page" n="27"/>
+               <ptr type="digitised" target="https://gallica.bnf.fr/ark:/12148/bpt6k33044x/f43.item"/>
+            </bibl>
+            <listWit>
+               <witness xml:id="L">
+                  <abbr type="siglum">L</abbr>
+                  <idno>HILDESHEIM,  Sankt Godohard Kirche,  Psautier de Saint-Alban</idno>
+                  <origDate>1101-1200</origDate>
+                  <origPlace>England (Saint-Albans)</origPlace>
+                  <ptr type="digitised" target="https://www.abdn.ac.uk/stalbanspsalter/english/translation/trans058.shtml"/>
+                  <ptr type="description" target="http://jonas.irht.cnrs.fr/manuscrit/24358"/>
+                  <!-- Mis la note dans un desc… note est refusé -->
+                  <desc>Famous manuscript, perhaps from the 1130's, one of the very rare french ms. for the first half of the XIIth century</desc>
```

Comment:
Weird, the Guidelines say it is possible:
https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-witness.html

```xml
<content>
 <alternate minOccurs="0"
  maxOccurs="unbounded">
  <textNode/>
  <classRef key="model.limitedPhrase"/>
  <classRef key="model.inter"/>
  <elementRef key="note"/>
  <elementRef key="object"/>
 </alternate>
</content>
```